frontend/ui/api/src/main/java/com/virtuslab/gitmachete/frontend/ui/services/GraphTableService.java

```diff
@@ -9,12 +9,12 @@
 import com.virtuslab.gitmachete.frontend.ui.api.table.IBaseEnhancedGraphTableFactory;
 
 @Service
```

Comment:
Hmmm `GitMacheteRepositoryCacheService` isn't marked with `@Service`... what's the difference? 🤔 Is it b/c of `<applicationService ...>` entry in plugin.xml? for consistency, is it possible to also register the existing `@Service`s in plugin.xml?

Reply:
Ah, yeah it could be marked with @Service, but since that annotation comes from IntelliJ we would need to add it to the backend classpath. We probably don't want that.